crates/miden-protocol-macros/src/lib.rs

```diff
@@ -141,7 +141,7 @@ pub fn word_wrapper_derive(input: TokenStream) -> TokenStream {
             ///
             /// This requires the caller to uphold the guarantees/invariants of this type (if any).
             /// Check the type-level documentation for guarantees/invariants.
-            pub fn new_unchecked(word: Word) -> Self {
+            pub fn from_raw(word: Word) -> Self {
```

Comment:
Seems like this is a change from another PR (and there are a few other such changes in this PR).